package.json

```diff
@@ -98,7 +98,7 @@
   "dependencies": {
     "@across-protocol/across-token": "^1.0.0",
     "@across-protocol/constants-v2": "^1.0.9",
-    "@across-protocol/contracts-v2": "^2.4.7",
+    "@across-protocol/contracts-v2": "^2.5.0-beta.2",
```

Comment:
One consideration: Doing this on master could leave us in a situation where we can't bump sdk-v2 despite needing to get a change into production. I wonder if it might be safer to have this on a development branch.

Reply:
Hmm